cstar/orchestration/models.py

```diff
@@ -358,6 +367,11 @@ def _model_validator(self) -> "RomsMarblBlueprint":
 
         return self
 
+    @property
+    def cpus_needed(self) -> int:
+        """Number of CPUS needed for ROMS is derived from the partitioning parameters."""
```

Comment:
super nit: don't capitalize `S` or it becomes part of the acronym.

This docstring feels like it could use a minor tweak. As-is, we state an interesting fact but it doesn't seem right. Consider just dropping "is" or using an extended description:

`Number of CPUs needed for ROMS (derived from partitioning parameters).`

or

```python
"""
Number of CPUs needed for ROMS.

The count is computed from partitioning parameters.
"""
```